src/pages/dashboard/feedstock/[id].js

```diff
@@ -79,36 +46,25 @@ const Feedstock = () => {
       </Layout>
     )
 
-  const name = spec
-    .replace('pangeo-forge/', '')
-    .replace(new RegExp('_', 'g'), '-')
-
+  const name = getName(spec)
   const { isProduction, datasets } = getProductionRunInfo(id, recipe_runs)
 
   return (
     <Layout>
       <Box as='section'>
         <Container maxW='container.xl' mt={90}>
-          <Button
-            as={Link}
-            href={repoUrl}
-            fontSize={'2xl'}
-            colorScheme='teal'
-            variant='outline'
-          >
-            {' '}
-            <GoRepo />
-            <Text ml={2}>{name}</Text>
-          </Button>
-
-          {spec == 'pangeo-forge/staged-recipes' && (
-            <Text my={4}>
-              A place to submit pangeo-forge recipes before they become fully
-              fledged pangeo-forge feedstocks.
-            </Text>
-          )}
-
-          <FeedstockInfo details={details} />
+          <FeedstockInfo
+            repo={spec}
+            name={name}
+            title={meta.title}
+            description={meta.description}
+            pangeo_forge_version={meta.pangeo_forge_version}
+            pangeo_notebook_version={meta.pangeo_notebook_version}
+            bakery={meta.bakery ? meta.bakery.id : null}
```

Comment:
[Optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) would help make this prop transfer slightly more concise!
```suggestion
            bakery={meta.bakery?.id}
```